iron-form-element-behavior.js

```diff
@@ -50,31 +37,11 @@ export const IronFormElementBehavior = {
     /**
      * Set to true to mark the input as required. If used in a form, a
      * custom element that uses this behavior should also use
-     * Polymer.IronValidatableBehavior and define a custom validation method.
+     * IronValidatableBehavior and define a custom validation method.
      * Otherwise, a `required` element will always be considered valid.
      * It's also strongly recommended to provide a visual style for the element
      * when its value is invalid.
      */
     required: {type: Boolean, value: false},
-
-    /**
-     * The form that the element is registered to.
-     */
-    _parentForm: {type: Object}
```

Comment:
I think `_parentForm` and empty `attached` and `detached` methods should stay in the definition in case someone's using them (weirdly).

Reply:
but it's a custom element, it already has attached and detached?

Reply:
ok